doc/releases/pacific.rst

```diff
@@ -29,6 +29,165 @@ giant pacific octopus (Enteroctopus dofleini).
    DO NOT run ``ceph-bluestore-tool``'s repair/quick-fix commands.
 
 
+v16.2.7 Pacific   
+===============
+
+This is the seventh backport release in the Pacific series.
+
+Notable Changes
+---------------
+
+* Critical bug in OMAP format upgrade is fixed. This could cause data corruption
+  (improperly formatted OMAP keys) after pre-Pacific cluster upgrade if
+  bluestore-quick-fix-on-mount parameter is set to true or ceph-bluestore-tool's
+  quick-fix/repair commands are invoked.
+  Relevant tracker: https://tracker.ceph.com/issues/53062
```

Comment:
bluestore-quick-fix-on-mount parameter is now safe, but should we remind users that it is still not recommended (and still not the default) ? @neha-ojha @ifed01

Reply:
@dvanders I am not sure want to say "not recommended" but I added a note about it not being set to true by default yet.